assets/js/index.js

```diff
@@ -0,0 +1,8 @@
+import { Carousel } from "./carousel.js"
+

```

Comment:
Se a lista de imagens for dinâmica vc pode passar para o construtor o caminho para uma pasta ou passar um array com os src das imagens